CHANGELOG.md

```diff
@@ -0,0 +1,314 @@
+# Fastlane CHANGELOG
+
```

Comment:
@lacostej The `Rakefle` I talk about in the PR description is one for another repo of mine (SwiftGen), not one in Fastlane yet 😉  fast lane obviously uses a Fastfile to automate its release process 😛 

So basically we'll just modify [fastlane's own `Fastfile` here](https://github.com/fastlane/fastlane/blob/master/fastlane/Fastfile#L88-L89), extracting the `description` from the `CHANGELOG.md` instead of prompting for it.
The code for extracting can indeed be inspired by my Rakefile but will have to be adapted for fastlane's needs indeed :wink: